js/layout-javascript/simple-list-code.js

```diff
@@ -480,6 +548,7 @@ DynamicList.prototype.attachObservers = function() {
     .on('click', '.simple-list-comment-close-panel', function() {
```

Comment:
Why does this not need `keydown` event? like the `.news-feed-comment-close-panel` selector needed?

Reply:
@tonytlwu, fixed